contrib/pg_tde/documentation/docs/setup.md

```diff
@@ -1,137 +1,69 @@
-
-        * `url` is the URL of the Vault server
-        * `mount` is the mount point where the keyring should store the keys
-        * `root_token` is an access token with read and write access to the above mount point
-        * [optional] `ca_path` is the path of the CA file used for SSL verification
-
-        <i warning>:material-information: Warning:</i> This example is for testing purposes only:
-
-        ```
-        SELECT pg_tde_add_global_key_provider_vault_v2('my-vault','http://vault.vault.svc.cluster.local:8200,'secret/data','hvs.zPuyktykA...example...ewUEnIRVaKoBzs2', NULL);
-        ```
-
-    === "With a keyring file"
-
-        This setup is intended for development and stores the keys unencrypted in the specified data file. See [how to use external reference to parameters](external-parameters.md) to add an extra security layer to your setup.
-  
-
-        ```sql
-        SELECT pg_tde_add_global_key_provider_file('provider-name','/path/to/the/keyring/data.file');
-        ```
-
-        <i warning>:material-information: Warning:</i> This example is for testing purposes only:
-
-        ```sql
-        SELECT pg_tde_add_global_key_provider_file('file-keyring','/tmp/pg_tde_test_local_keyring.per');
-        ```
-       
-       
-2. Add a default principal key
-
-    ```sql
-    SELECT pg_tde_set_default_key_using_global_key_provider('name-of-the-key','provider-name','ensure_new_key');
-    ```
+```
+    CREATE EXTENSION pg_tde;
+```
 
-    where:
+!!! note
+    The `pg_tde` extension is created only for the current database. To enable it for other databases, you must run the command in each individual database.
 
-    * `name-of-the-key` is the name of the principal key. You will use this name to identify the key.
-    * `provider-name` is the name of the key provider you added before. The principal key will be associated with this provider.
-    * `ensure_new_key` defines if a principal key must be unique. The default value `true` means that you must speficy a unique key during key rotation. The `false` value allows reusing an existing principal key.
+## 4. (Optional) Enable `pg_tde` by default for new databases
 
-    <i warning>:material-information: Warning:</i> This example is for testing purposes only. Replace the key name and provider name with your values:
+To automatically have `pg_tde` enabled for all new databases, modify the `template1` database:
 
-    ```sql
-    SELECT pg_tde_set_key_using_global_key_provider('test-db-master-key','file-vault','ensure_new_key');
-    ```
-
-    The key is auto-generated.
+```
+    psql -d template1 -c 'CREATE EXTENSION pg_tde;'
+```
 
-    After this, all databases that do not have something else configured will use this newly generated principal key.
+!!! note
+    You can use external key providers to manage encryption keys. The recommended approach is to use the Key Management Store (KMS). See the next step on how to configure the KMS.
 
-## Next steps
+## Next step
```

Comment:
Again, here you use sentence style. On other pages you have "Next **S**tep". Consider using the same style everywhere